src/main/java/org/openelisglobal/odoo/config/TestProductMapping.java

```diff
@@ -0,0 +1,143 @@
+package org.openelisglobal.odoo.config;
+
+import jakarta.annotation.PostConstruct;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import lombok.Getter;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.openelisglobal.common.log.LogEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TestProductMapping {
+
+    private static final String ENV_MAPPING_FILE = "ODOO_MAPPING_FILE";
+    private static final String DEFAULT_CSV_FILE = "odoo-test-product-mapping.csv";
+    private final Map<String, TestProductInfo> testToProductInfo = new HashMap<>();
+
+    @Getter
+    public static class TestProductInfo {
+        private final String productName;
+        private final double priceUnit;
+        private final double quantity;
+
+        public TestProductInfo(String productName, double quantity, double priceUnit) {
+            this.productName = productName;
+            this.quantity = quantity;
+            this.priceUnit = priceUnit;
+        }
+    }
+
+    @PostConstruct
+    public void init() {
+        boolean loaded = loadCsvFromEnvironment() || loadCsvFromClasspath();
+        int mappingsLoaded;
+        if (loaded) {
+            mappingsLoaded = testToProductInfo.size();
+            LogEvent.logInfo(getClass().getSimpleName(), "init",
+                    "Loaded CSV mapping. Total mappings loaded: " + mappingsLoaded);
+        } else {
+            LogEvent.logError(getClass().getSimpleName(), "init",
+                    "No CSV mapping file could be loaded (environment or classpath).");
+            return;
+        }
+
+        if (mappingsLoaded == 0) {
+            LogEvent.logWarn(getClass().getSimpleName(), "init", "No valid mappings found.");
+        } else {
+            LogEvent.logInfo(getClass().getSimpleName(), "init",
+                    "Available mapping keys: " + String.join(", ", getAllMappedTestCodes()));
+        }
+    }
+
+    private boolean loadCsvFromEnvironment() {
+        String path = System.getenv(ENV_MAPPING_FILE);
```

Comment:
just use a fixed path here ie `/var/lib/openelis-global/odoo/odoo-test-product-mapping.csv`,
add the default file under the volumes and just mount it to that directory in docker compose

Reply:
Users can always modify that file and mount it into the container